search-commons/src/main/java/no/unit/nva/search/constants/ApplicationConstants.java

```diff
@@ -63,4 +70,37 @@ private static String readSearchInfrastructureApiUri() {
     private static String readSearchInfrastructureAuthUri() {
         return ENVIRONMENT.readEnv("SEARCH_INFRASTRUCTURE_AUTH_URI");
     }
+
+
+    private static TermsAggregationBuilder generateSimpleAggregation(String term, String field) {
+        return AggregationBuilders
+                .terms(term)
+                .field(field)
+                .size(100);
```

Comment:
100 should be a constant so its easier to understand what this magic number means. NUMBER_OF_AGGREGATED_BUCKETS, or something